### PR TITLE
minor doc fixes for release

### PR DESCRIPTION
**Pull request by andrei-stoian-zama**
Closes https://github.com/zama-ai/concrete-ml-internal/issues/3974
Closes https://github.com/zama-ai/concrete-ml-internal/issues/3972
Closes https://github.com/zama-ai/concrete-ml-internal/issues/4020
Closes https://github.com/zama-ai/concrete-ml-internal/issues/3973

### CODE REVIEW

File: docs/built-in-models/linear.md [hunk 1]
```diff
@@ -80,3 +82,21 @@ We can then plot the decision boundary of the classifier and compare those resul
...
+```
+from sklearn import LogisticRegression as SKlearnLogisticRegression
+
+# Instantiate the model:
+model = SKlearnLogisticRegression()
+
+# Fit the model:
+model.fit(X_train, y_train)
+
+cml_model = LogisticRegression(model, X_train, n_bits=8)
```
Comment (RomanBredehoft) [inline docs/built-in-models/linear.md:101]: it's `LogisticRegression.from_sklearn_model` !

File: docs/built-in-models/linear.md [hunk 2]
```diff
@@ -80,3 +82,21 @@ We can then plot the decision boundary of the classifier and compare those resul
...
+```
```
Comment (RomanBredehoft) [inline docs/built-in-models/linear.md:102]: also maybe I would add the compile + predict steps to better show that this does not replace other steps (like the compile one) , as you wish

**Review by RomanBredehoft — CHANGES_REQUESTED** [review]
thanks a lot for this ! I have a few comments

File: README.md [hunk 3]
```diff
@@ -117,28 +117,28 @@ Full, comprehensive documentation is available here: [https://docs.zama.ai/concr
...
 
-- [FHE neural network splitting for client/server deployment](use_case_examples/cifar/cifar_brevitas_with_model_splitting): we explain how to split a computationally-intensive neural network model in two parts. First, we execute the first part on the client side in the clear, and the output of this step is encrypted. Next, to complete the computation, the second part of the model is evaluated with FHE. This tutorial also shows the impact of FHE speed/accuracy trade-off on CIFAR10, limiting PBS to 8-bit, and thus achieving 62% accuracy.
+- [FHE neural network splitting for client/server deployment](use_case_examples/cifar/cifar_brevitas_with_model_splitting): explains how to split a computationally-intensive neural network model in two parts. First, we execute the first part on the client side in the clear, and the output of this step is encrypted. Next, to complete the computation, the second part of the model is evaluated with FHE. This tutorial also shows the impact of FHE speed/accuracy trade-off on CIFAR10, limiting PBS to 8-bit, and thus achieving 62% accuracy.
```
Comment (kcelia) [inline README.md:137]: explain (like you did above) or explains ?

File: docs/developer-guide/api/README.md [hunk 4]
```diff
@@ -240,7 +243,6 @@
 - [`utils.is_regressor_or_partial_regressor`](./concrete.ml.common.utils.md#function-is_regressor_or_partial_regressor): Indicate if the model class represents a regressor.
 - [`utils.manage_parameters_for_pbs_errors`](./concrete.ml.common.utils.md#function-manage_parameters_for_pbs_errors): Return (p_error, global_p_error) that we want to give to Concrete.
 - [`utils.replace_invalid_arg_name_chars`](./concrete.ml.common.utils.md#function-replace_invalid_arg_name_chars): Sanitize arg_name, replacing invalid chars by \_.
-- [`utils.set_multi_parameter_in_configuration`](./concrete.ml.common.utils.md#function-set_multi_parameter_in_configuration): Build a Configuration instance with multi-parameter strategy, unless one is already given.
```
Comment (kcelia) [inline docs/developer-guide/api/README.md:243]: have you meant to delete mono ?  [`utils.force_mono_parameter_in_configuration`](./concrete.ml.common.utils.md#function-force_mono_parameter_in_configuration): Force configuration to mono-parameter strategy.

File: docs/developer-guide/api/concrete.ml.deployment.fhe_client_server.md [hunk 5]
```diff
@@ -14,11 +14,33 @@ ______________________________________________________________________
...
+check_concrete_versions(zip_path: Path)
+```
+
+Check that current versions match the ones used in development.
+
+This function loads the version JSON file found in client.zip or server.zip files and then checks that current package versions (Concrete Python, Concrete ML) as well as the Python current version all match the ones that are currently installed.
+
+**Args:**
+
+- <b>`zip_path`</b> (Path):  The path to the client or server zip file that contains the version.json  file to check.
```
Comment (kcelia) [inline docs/developer-guide/api/concrete.ml.deployment.fhe_client_server.md:29]: extra space in `(Path):  The `

File: docs/developer-guide/api/concrete.ml.deployment.fhe_client_server.md [hunk 6]
```diff
@@ -14,11 +14,33 @@ ______________________________________________________________________
...
+
+**Raises:**
+
+- <b>`ValueError`</b>:  If at least one version mismatch is found.
```
Comment (kcelia) [inline docs/developer-guide/api/concrete.ml.deployment.fhe_client_server.md:33]: extra space in `</b>:  If `

**Review by fd0r — APPROVED** [review]
Good with me, but @RomanBredehoft 's comment wasn't addressed

**Review by RomanBredehoft — APPROVED** [review]

**Pull request merged**